src/style.css

```diff
@@ -1 +1,30 @@
-/* add styles here */
+.page__budy {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-size: cover;
+  background-position: center;
```

Comment:
You are using a tag name for styling, which is not recommended. It's better to style all elements using `.classes` and if needed with `:pseudo-classes`, `pseudo-elements` and `[attributes]`.